test/e2e/ccip/LooperFork.t.sol

```diff
@@ -0,0 +1,158 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+import {Test, Vm} from "forge-std/Test.sol";
+import {CCIPLocalSimulatorFork, Register} from "../../../src/ccip/CCIPLocalSimulatorFork.sol";
+
+import {IRouterClient} from "@chainlink/contracts-ccip/contracts/interfaces/IRouterClient.sol";
+import {Client} from "@chainlink/contracts-ccip/contracts/libraries/Client.sol";
+import {CCIPReceiver} from "@chainlink/contracts-ccip/contracts/applications/CCIPReceiver.sol";
+import {IERC20} from
+    "@chainlink/contracts/src/v0.8/vendor/openzeppelin-solidity/v4.8.3/contracts/token/ERC20/IERC20.sol";
+
+contract Looper is CCIPReceiver {
```

Comment:
Nice test!
Only feedback here is "Looper" not very clear or domain-specific. It makes sense to us because we already know what it does, but to someone else it can be confusing.
 A better name for the Contract/Object and also this test file would reflect its role as a CCIP bulk/multi message sender & receiver for testing or simulation.  
 
something like MultiSender or something?  Have a think!